### ISSUE TITLE
isinstance not working with JS arrays

### ISSUE THREAD

**Issue by denis-migdal** [brython-dev/brython#2325]
`isinstance(arr, list)`/`isinstance(arr, tuple)` doesn't work properly on JS arrays (one of them should be `true`.

Maybe will be fixed by the PR I made a few time back.

**Comment by PierreQuentel** [comment]
I don't agree that either test should return `True`, a Javascript Array is not the same type as a Python list or tuple
```python
from browser import window

t = window.js_list
print(hasattr(t, 'push')) # True

t = [1]
print(hasattr(t, 'push')) # False
```

**Comment by denis-migdal** [comment]
Shouldn't they convert to one in `jsobj2pyobj` ?

**Comment by denis-migdal** [comment]
Or to be put in a wrapper `JSList` inheriting from `List`, or `JSTuple` inheriting from `Tuple`, depending on what `Object.isFrozen(jsarray)` returns ?
